pkg/services/sqlstore/dashboard.go

```diff
@@ -171,6 +183,11 @@ func SearchDashboards(query *search.FindPersistedDashboardsQuery) error {
 		params = append(params, "%"+query.Title+"%")
 	}
 
+  if len(query.Tags) > 0 {
+    sql.WriteString(" GROUP BY dashboard.slug HAVING COUNT(dashboard.slug) = ?")
```

Comment:
what is this doing here? And wont this require aggregation functions for fields in the select clause, and cause issue with the tag outer join so we do not get all the tags any more?